4 reports.sql

```diff
@@ -0,0 +1,42 @@
+--1) I want to know which personality types are most likely to pay for the report. Show which 4-letter types have bought the most full reports
```

Comment:
I want the three biggest buyers, and I want to know how many were bought PER personality type.